spec/delayed/worker_spec.rb

```diff
@@ -59,12 +59,15 @@
       Delayed::Worker.read_ahead = @read_ahead
     end
 
-    it 'reads five jobs' do
+    xit 'reads five jobs' do
       expect(Delayed::Job).to receive(:find_available).with(anything, 5, anything).and_return([])
+      p "##################"
+      p "##################"
+      p "##################"
```

Comment:
I think I'll need to pair up here. Not sure if we'd expect the read ahead stuff to still work when using the active record back end.

Reply:
Not using this so just kill it

Reply:
Ditch the code as well (hard return read ahead as 1).

Reply:
Remove read_ahead setter and always return 1 (since we're not using it)